plaidml/edsl/tests/edsl_test.cc

```diff
@@ -1467,6 +1467,56 @@ TEST_F(CppEdsl, Gather) {
   checkExact(program, {in1, in2}, {out});
 }
 
+TEST_F(CppEdsl, GatherTmp) {
```

Comment:
Test name 'Tmp' stands for temporary?

Reply:
Yes this is mostly a draft for early review. I will change it later.